### PR TITLE
Remove trailing slash from SSE configuration

### PR DESCRIPTION
**Pull request by kzantow**
**Decription**

Bumping versions, cleanup. This is a placeholder for the cleanup from upstream: https://github.com/jenkinsci/sse-gateway-plugin/pull/6

**Submitter checklist**
- [x] Change is code complete and matches issue description
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

**Reviewer checklist**
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explaination given

@reviewbybees

### CODE REVIEW

**Comment by tfennelly** [comment]
@kzantow is this PR dead?

What issue is this fixing anyway ? .. I remember you did another PR on the SSE Gateway, but I'm not aware of anything not working anywhere.

**Comment by kzantow** [comment]
@tfennelly no, it should be changed (it's such a minor thing) but it aligns with Jenkins proper and BO, just need to coordinate updating BO when the new version is published

File: blueocean-dashboard/src/main/js/Dashboard.jsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ appConfig.loadConfig();
 sse.connect({
     clientId: 'jenkins_blueocean',
     onConnect: undefined,
-    jenkinsUrl: `${appConfig.getJenkinsRootURL()}/`, // FIXME sse should not require this to end with a /
```
Comment (kzantow) [inline blueocean-dashboard/src/main/js/Dashboard.jsx]: @tfennelly the purpose of this PR is to fix this hack, but needs the corresponding change in SSE, which hasn't been merged AFAIK

**Comment by tfennelly** [comment]
@kzantow there's a `1.9-SNAPSHOT` available with your change (and another) if you want to try it against this PR.

File: pom.xml [hunk 2]
```diff
@@ -190,7 +190,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>sse-gateway</artifactId>
-            <version>1.5</version>
+            <version>1.6</version>
```
Comment (tfennelly) [inline pom.xml]: Update to `1.9-SNAPSHOT`.

File: blueocean-dashboard/package.json [hunk 3]
```diff
@@ -38,7 +38,7 @@
     "@jenkins-cd/design-language": "0.0.65",
     "@jenkins-cd/js-extensions": "0.0.19",
     "@jenkins-cd/js-modules": "0.0.5",
-    "@jenkins-cd/sse-gateway": "0.0.6",
+    "@jenkins-cd/sse-gateway": "0.0.7",
```
Comment (tfennelly) [inline blueocean-dashboard/package.json]: Update this to `0.0.8-beta1`

**Comment by tfennelly** [comment]
@kzantow see above for the version numbers to update to.

**Comment by michaelneale** [comment]
closing this as its quite old... keith has his branch if he needs to find it

**Pull request closed without merging**